model/src/main/kotlin/OrtResult.kt

```diff
@@ -69,7 +69,7 @@ data class OrtResult(
      * which are customizable by the user, for example in evaluator rules or in the notice reporter.
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val labels: Map<String, String> = emptyMap()
+    val labels: SortedSet<String> = sortedSetOf()
```

Comment:
Just thinking out aloud: Should this be a plain `Set`? I.e., do we benefit from ordering here? Lookup would be after, but insert slower, but I guess for the expected small number of labels we do not care either way.

Reply:
`Set` is immutable as opposed to the `sorted` one.

Reply:
Right, sorry, I was of course meaning `MutableSet`.

Reply:
It's only for serialization, see the commit message. And yes, performance should not matter here, even if you have hundreds of labels which is not expected.

Reply:
So, does this actually *have* to mutable?